application/src/main/java/net/corda/v5/application/crypto/SigningServiceSignContext.java

```diff
@@ -0,0 +1,18 @@
+package net.corda.v5.application.crypto;
+
+import net.corda.v5.base.annotations.CordaSerializable;
+
+/*
+ * Context attached to the sign operation of SigningService
+ */
+@CordaSerializable
+public final class SigningServiceSignContext {
+    private final String keyCategory;
+
+    /**
+     * @param keyCategory the category of the key to be signed with
+     */
+    public SigningServiceSignContext(String keyCategory) {
```

Comment:
I suppose if ever want to add additional fields in the future we can add additional public constructors in the future without breaking compatibility. However, I wonder if it'd be simpler to specify a parameter directly in an overloaded method.

Reply:
It's just a minor observation, not a blocker.